src/test/java/hw2/ex2/Exercise2.java

```diff
@@ -0,0 +1,83 @@
+package hw2.ex2;
+
+import hw2.base.BaseClass;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.*;
+
+public class Exercise2 extends BaseClass {
+
+    @Test
+    public void exerciseTwoTest() {
+
+        // 1. Open test site by URL
+        driver.get("https://jdi-testing.github.io/jdi-light/index.html");
+        String actualTitle = driver.getTitle();
+        String expectedTitle = "Home Page";
+
+        // 2. Assert Browser title
+        assertEquals(actualTitle, expectedTitle);
+
+        // 3. Perform login
+        driver.findElement(By.id("user-icon")).click();
+        driver.findElement(By.id("name")).sendKeys("Roman");
+        driver.findElement(By.id("password")).sendKeys("Jdi1234");
+        driver.findElement(By.id("login-button")).click();
+
+        WebElement loginUserName = driver.findElement(By.id("user-name"));
+
+        // 4. Assert User name in the left-top side of screen that user is loggined
+        assertEquals(loginUserName.getText(), "ROMAN IOVLEV");
+
+        // 5. Open through the header menu Service -> Different Elements Page
+        WebElement serviceButton = driver.findElement(By.className("dropdown-toggle"));
+        serviceButton.click();
+
+        WebElement differentElementsPage = driver.findElement(By
+                .cssSelector("a[href='different-elements.html']"));
```

Comment:
Why do you decide use `href` attribute here in the selector?

Reply:
Simplified using By.linkText